xamarin/Xamarin.Forms
# [Android] Set the elevation hierarchy after the first layout cycle of the parent

### Description of Change ###
When Android first renderer a layout the elevation on the elements gets set after they are added to the visual tree. This doesn't appear to be the case once elements are added and removed after the app is already running. This PR iterates through the children after the parent layout has finished and just makes sure the elevations are all correct so that everything appears correctly

### Issues Resolved ### 
- fixes #6262
- fixes partially  #5642

### Testing Procedure ###
There's a UI test for this. If you want to run 6262 yourself make sure to set it up so the fast renderers are being used as this is only an issue there 

### PR Checklist ###
